emacs/radian.el

```diff
@@ -1007,16 +1006,17 @@ ourselves."
 
   ;; The default settings seem a little forgetful to me. Let's try
   ;; this out.
-  (setq prescient-history-length 1000))
+  (setq prescient-history-length 1000)
 
-;; Package `selectrum-prescient' provides intelligent sorting and
-;; filtering for candidates in Selectrum menus.
-(radian-use-package selectrum-prescient
-  :demand t
-  :after selectrum
-  :config
+  ;; Use prescient.el everywhere.
+  (setq completion-styles '(prescient))
```

Comment:
Following what was said on the Orderless and Vertico pages, it's probably best to include `basic` after `prescient` in this list. Some dynamic completion tables seem to assume `basic`, or only generate candidates after being passed a non-empty prefix string, which `prescient` and `orderless` don't do.

This is what I'm doing right now:

```elisp
(setq completion-styles '(prescient basic)
      completion-category-overrides '((file  (styles basic partial-completion))
                                      (eglot (styles prescient))))              

```